spinetoolbox/config.py

```diff
@@ -23,9 +23,9 @@
 # Invalid characters for directory names
 # NOTE: "." is actually valid in a directory name but this is
 # to prevent the user from creating directories like /..../
-INVALID_CHARS = ["<", ">", ":", "\"", "/", "\\", "|", "?", "*", "."]
+INVALID_CHARS = ["<", ">", ":", '"', "/", "\\", "|", "?", "*", "."]
 # Invalid characters for file names
-INVALID_FILENAME_CHARS = ["<", ">", ":", "\"", "/", "\\", "|", "?", "*"]
+INVALID_FILENAME_CHARS = ["<", ">", ":", '"', "/", "\\", "|", "?", "*"]
```

Comment:
@soininen This change looks suspicious to me...

Reply:
It is the "pythonic" way :) Works fine, by the way.